core/shared/src/main/scala/monocle/Fold.scala

```diff
@@ -59,42 +59,42 @@ trait Fold[A, B] { self =>
         self.toIterator(from).flatMap(other.toIterator)
     }
 
-  def asTarget[C](implicit ev: B =:= C): Fold[A, C] =
+  def asTarget[C](implicit ev: B <:< C): Fold[A, C] =
```

Comment:
Hmm that's interesting, why not `[C >: B]`?

Reply:
yes, it is better!